frontend/tests/acceptance/pages/signupPage.js

```diff
@@ -3,17 +3,46 @@ const { I } = inject();
 module.exports = {
   url: '/signup',
   fields: {
-    name: '//label[contains(text(),"name")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-    email: '//label[contains(text(),"email")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-    password: '//label[contains(text(),"password")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-    confirmPassword: '//label[contains(text(),"confirm password")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-  },
-  signUp(name, email, password, confirmPassword) {
-    I.waitForVisible(this.fields.name);
-    I.fillField(this.fields.name, name);
-    I.fillField(this.fields.email, email);
-    I.fillField(this.fields.password, password);
-    I.fillField(this.fields.confirmPassword, confirmPassword);
-    I.click('Sign Up');
+    username: '//label[contains(text(),"Name")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
+    email: '//label[contains(text(),"Email")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
+    password: '//label[contains(text(),"Password")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
+    confirmPassword: '//label[contains(text(),"Confirm password")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
   },
+  elements: {
+    error_lbl: '//*[@id="root"]/div/div/div/div/form/p[2]',
```

Comment:
```suggestion
    error_lbl: '//*[@id="root"]/div//form/p[2]',
```
I think this should work too

Reply:
maile inspection bata copy paste gareko, mero wala nachalera.. haha
let me try to redefine.